OKTA-459416 Custom Self-Service Pwd Recovery Guide for Java (#3292)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/beforeyoubegin.md

1. Build an app with the Embedded Sign-In Widget or SDK.
    * For embedded Sign-In Widget based apps, implement the [basic sign-in flow](/docs/guides/oie-embedded-widget-use-case-basic-sign-in/aspnet/main/).

>     * For SDK-based apps, implement [user password recovery](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/aspnet/main/).

Author: Thank you

> If you want to test the flows without building an app, use the provided sample apps. They have prebuilt support for password recovery email customizations.

Author: Subtle. Thank you.

> If you want to test the flows without building an app, use the provided sample apps. They have prebuilt support for password recovery email customizations.

Reviewer: How about a little simpler:

```suggestion
To test the flows without building an app, use one of the sample apps which include support for password recovery email customizations.
```

Author: fixed
